onnxsim/__main__.py

```diff
@@ -43,12 +49,33 @@ def main():
                 # for the input name like input:0
                 name, shape = ':'.join(
                     pieces[:-1]), list(map(int, pieces[-1].split(',')))
-                input_shapes[name] = shape
+                input_shapes.update({name: shape})
+
+    input_datas = dict()
+    if args.input_data is not None:
+        for x in args.input_data:
+            pieces = x.split(':')
+            name, data = pieces[0], pieces[-1]
+            input_datas.update({name: data})
+
+    input_tensors = dict()
+    if args.input_data is not None and atgs.input_shape is not None:
```

Comment:
`args.input_data is not None` -> `len(input_data_paths) > 0`

`atgs` is a typo